src/main/java/com/geoxp/geo/HHCodeHelper.java

```diff
@@ -437,14 +437,14 @@ public static final void stableSplitHHCode(final long hhcode, final int resoluti
    * @return
    */
   public static final long getCenter(long hhcode, int resolution) {
-    // Limit 'hhcode' to 'resolution'
-    if (resolution >= 2 && resolution <= 32) {
-      hhcode = hhcode & Coverage.PREFIX_MASK[(resolution >>> 1) - 1];
-    }
-
-    // Add the 'center bits' for the resolution just above 'resolution'
-    if (resolution >= 2 && resolution < 32) {
-      hhcode = hhcode | Coverage.CENTER_BITS[((resolution >>> 1) - 1) + 1];
+    if (0 == resolution) {
```

Comment:
What is the semantics of resolution 0? It is not used anywhere else.

Reply:
True, but I found it logical that a 0 resolution cell represents the whole planisphere. Updated the javadoc accordingly.